src/main/java/com/example/bookstore/validation/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,29 @@
+package com.example.bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator implements ConstraintValidator<FieldsMatch, Object> {
+    private String field;
+    private String fieldMatch;
+
+    public void initialize(FieldsMatch constraintAnnotation) {
```

Comment:
minor

add `@Override` to explicitly declare that this method is overridden from parent